index.bs

```diff
@@ -1394,21 +1404,38 @@ To <dfn>parse filter data</dfn> given a |value|:
 Issue: Determine whether to limit [=string/length=] or
 [=string/code point length=] for |filter| and |d| above.
 
+To <dfn>parse filter config</dfn> given a |value|:
+
+1. If |value| is not a [=map=], return null.
+1. Let |lookbackWindow| be null.
+1. If |value|["`_lookback_window`"] [=map/exists=]:
```

Comment:
Do we want to take this opportunity to reserve all keys beginning with `_`, or at least add an issue in the spec to do so? That would help simplify future changes similar to this one.

Reply:
Good idea, I'll add this in a follow up PR. I added **https://github.com/WICG/attribution-reporting-api/issues/930**.